src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -57,7 +57,8 @@ public enum Option
         ADDITIONAL_WRITE_POLICY,
         CRC_CHECK_CHANCE,
         CDC,
-        READ_REPAIR;
+        READ_REPAIR,
+        DISABLED_AUTOMATED_REPAIR;
```

Comment:
NIT: from a naming, having `AUTOMATED_REPAIR: true` vs `DISABLED_AUTOMATED_REPAIR: false` is easier for people to reason. Avoiding the double negatives and extra boolean logic for what people expect.

Reply:
ACK - will update it accordingly

Reply:
Done